Source/WTF/wtf/PlatformHave.h

```diff
@@ -1512,3 +1559,27 @@
 #if !PLATFORM(VISION)
 #define HAVE_IDLE_SLEEP_STATE 1
 #endif
+
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
+    || (PLATFORM(MACCATALYST) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
+    || PLATFORM(IOS) \
```

Comment:
This looks like IOS_FAMILY to me?

Reply:
I ended up needing to keep this separate for now, since the different IOS_FAMILY things are gated on different versions, and failing to do so broke the Open Source builders.